config/environments/production.rb

```diff
@@ -79,19 +79,19 @@
     address: 'smtp.mailgun.org',
     port: 587,
     authentication: :plain,
-    user_name: 'postmaster@speeduplouisville.com',
-    password: '740629340bec07c903fca9d96e35325b',
-    domain: 'speeduplouisville.com',
+    user_name: 'postmaster@speedupamerica.com',
```

Comment:
Thanks. Good catch on these, I'm not sure why someone would check in SMTP credentials into a repo.